init.sql

```diff
@@ -0,0 +1,103 @@
+CREATE TABLE IF NOT EXISTS users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    role VARCHAR(50),
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+    );
+
+INSERT INTO users (email, password, role) VALUES
+('admin@example.com', 'password', 'admin'),
+('user@example.com', 'password', 'user');
+
+CREATE TABLE IF NOT EXISTS accounts (
+    id SERIAL PRIMARY KEY,
+    user_id INTEGER UNIQUE NOT NULL,
+    first_name VARCHAR(255),
+    last_name VARCHAR(255),
+    prefered_name VARCHAR(255),
+    date_of_birth DATE,
+    phone VARCHAR(20),
+    gender VARCHAR(10),
+    photo_url VARCHAR(1024),
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT fk_user
+        FOREIGN KEY (user_id)
+            REFERENCES users (id)
+            ON DELETE CASCADE
+);
+
+INSERT INTO accounts (user_id, first_name, last_name, prefered_name, date_of_birth, phone, gender, photo_url) VALUES
+(1, 'Paul', 'McCartney', 'Super Paul', '1990-05-15', '123456789', 'M', 'http://example.com/photo1.jpg'),
+(2, 'John', 'Lennon', 'Super John', '1990-05-05', '987654321', 'M', 'http://example.com/photo2.jpg');
+--
```

Comment:
How it should work, if you didn't set user_id, but it has constraint 'UNIQUE NOT NULL'?

Reply:
Not clear for me. Should I some how add SELECT script to get ids from user table?